jama5262/Politico
# #163832115 Politico fixed all the minor duplicate codes from code climate

**What does this PR do?**
Fix minor duplicated found by code climate

**Description of Task to be completed?**

1. Fix and implement dry for minor

**How should this be manually tested?**
Go to [Gitbut Page](https://github.com/jama5262/Politico), make sure you are on the develop branch, in the readMe file there is a code climate build badge, click on it to view more info

**What are the relevant pivotal tracker stories?**
[#163832115 ](https://www.pivotaltracker.com/n/projects/2241896)
